crates/ruff_linter/src/checkers/logical_lines.rs

```diff
@@ -86,7 +86,7 @@ pub(crate) fn check_logical_lines(
 
         let indent_level = expand_indent(locator.slice(range));
 
-        let indent_size = 4;
+        let indent_size = settings.tab_size.as_usize();
```

Comment:
What happens if the user indents using tabs? What _should_ happen?

Reply:
This should be `indent_width` instead of `tab_size`, right?

Reply:
`LinterSettings.tab_size` is inferred from `Settings.indent_width`.

Reply:
Ah okay, I thought that it was the tab size from the settings 😄